### ISSUE TITLE
Loading MathJax with SVG component throws exception in browser

### ISSUE THREAD

**Issue by kepempem** [mathjax/MathJax#2408]
### Issue Summary

I'm trying to use the **_tex-svg-full_** component (And I'm willing to use any other component, I just want the output to be svg). I also tried some other components that use SVG and the same happens with them. Loading MathJax or simply trying to typeset throws the following error in some of the pages in my site:
```
tex-svg-full.js:formatted:16622 Uncaught (in promise) TypeError: t.setAttribute is not a function
    at s.setAttribute (tex-svg-full.js:formatted:16622)
    at e.assistiveMml (tex-svg-full.js:formatted:16337)
    at e.assistiveMml (tex-svg-full.js:formatted:16078)
    at Q.assistiveMml (tex-svg-full.js:formatted:16361)
    at Object.renderDoc (tex-svg-full.js:formatted:10486)
    at L.renderDoc (tex-svg-full.js:formatted:10502)
    at Q.w.render (tex-svg-full.js:formatted:10654)
    at tex-svg-full.js:formatted:38255
    at e (tex-svg-full.js:formatted:14082)
    at new Promise (<anonymous>)
```

This is how the origin of the error looks in MathJax:
```
s.prototype.setAttribute = function(t, e, Q, T) {
        return void 0 === T && (T = null),
        T ? (e = T.replace(/.*\//, "") + ":" + e.replace(/^.*:/, ""),
        t.setAttributeNS(T, e, Q)) : t.setAttribute(e, Q)
    }
```
Where ``t.setAttribute(e, Q)`` is the cause for the error.

### Technical details:

* MathJax Version: 3
* Client OS: Mac OS X 10.8.4
* Browser: Chrome 29.0.1547.57

### Supporting information:

This is my website: https://mathematics.rocks/
And for some reason, in some pages, the error is thrown while in others everything works just fine.

This is how I'm loading MathJax:
```
window.MathJax = this.config.MathJaxConfig;
var script = document.createElement("script");
script.src = "https://cdn.jsdelivr.net/npm/mathjax@3/es5/"+this.config.MathJaxComponent+".js";
script.async = true;
document.head.appendChild(script);
```
where `this.config.MathJaxComponent = "tex-svg-full"`

This is how I'm typesetting:
```
MathJax.typesetClear();
MathJax.texReset();
MathJax.typeset();
```

When I used the _**tex-mml-chtml**_ component everything worked perfectly but I needed to switch to SVG.

**Comment by dpvc** [comment]
This turns out to be the same issue as #2402 (you are using under- and over-braces, which are causing the trouble; other characters can also trigger the problem).

The work-around listed in that issue should resolve the problem for you.

**Comment by kepempem** [comment]
@dpvc I tried using the workaround but the same exception is still thrown. I'm not using any \overbrace in most pages, maybe in one of them. Can you please tell me what is causing the error? For example, this page: https://mathematics.rocks/In-The-Beginning/Notation has no overbraces or any sort of such things and yet it fails to load.

**Comment by dpvc** [comment]
I've modified the code slightly to change

``` javascript
              if (!variant) variant = this.variant;
```

 to

``` javascript
              if (!variant) variant = this.variant || 'normal';
```

I believe that will take care of it.

**Comment by kepempem** [comment]
@dpvc This fixed it! Can't thank you enough, you really saved me!

**Comment by dpvc** [comment]
Since I introduced the error in 3.0.5, I figure I had better help out those who are affected by it.

Glad it worked.

**Comment by dpvc** [comment]
Fixed in v3.1 released today.
